__tests__/integration/configuration/docker-compose.yml

```
@@ -29,12 +29,18 @@ services:
         target: /privnet/conf
     networks:
       - hathor-privnet
+    healthcheck:
+      test: ["CMD", "python", "-c", "import urllib.request; urllib.request.urlopen('http://localhost:8080/v1a/status')"]
```

suggestion(non-blocking): Maybe we should check the server state inside the status.

```suggestion
      test: ["CMD", "python", "-c", "import urllib.request; import json; r = urllib.request.urlopen('http://localhost:8080/v1a/status'); body = json.loads(r.read()); assert body['server']['state'] == 'READY'"]
```

This would make the healthcheck fail if the status is not a JSON object, invalid or if the fullnode reports a state other than `READY`.
What do you think?

---

Makes sense, even better. Done https://github.com/HathorNetwork/hathor-wallet-lib/pull/819/commits/de702e9590617113d676699850d1f4dcb7e6de85